pkgs/top-level/python-packages.nix

```diff
@@ -7242,6 +7244,8 @@ self: super: with self; {
 
   pcpp = callPackage ../development/python-modules/pcpp { };
 
+  pdb2pqr = callPackage ../development/python-modules/pdb2pqr { };
```

Comment:
Has this a lib part? Otherwise this entry should be removed.

Reply:
Yes, `pdb2pqr` also has a python interface that can be extended.